content/id/docs/setup/production-environment/container-runtimes.md

```diff
@@ -0,0 +1,414 @@
+---
+title: Runtime Container
+content_type: concept
+weight: 10
+---
+<!-- overview -->
+{{< feature-state for_k8s_version="v1.6" state="stable" >}}
+
+Untuk menjalankan Container di Pod, Kubernetes menggunakan _runtime_ Container (Container runtimes). Berikut ini adalah
+petunjuk instalasi untuk berbagai macam _runtime_.
+
+
+<!-- body -->
+
+
+{{< caution >}}
+Sebuah kekurangan ditemukan dalam cara `runc` menangani pendeskripsi berkas (_file_) sistem ketika menjalankan Container.
+Container yang berbahaya dapat menggunakan kekurangan ini untuk menimpa konten biner `runc` dan
+akibatnya Container tersebut dapat menjalankan perintah yang sewenang-wenang pada sistem host dari Container tersebut.
+
+Silahkan merujuk pada [CVE-2019-5736](https://access.redhat.com/security/cve/cve-2019-5736) untuk informasi lebih lanjut tentang masalah ini.
+{{< /caution >}}
+
+### Penerapan
+
+{{< note >}}
+Dokumen ini ditulis untuk pengguna yang memasang CRI (Container Runtime Interface) pada sistem operasi Linux. Untuk sistem operasi yang lain, 
+silahkan cari dokumentasi khusus untuk platform kamu.
+
+{{< /note >}}
+
+Kamu harus menjalankan semua perintah dalam panduan ini sebagai `root`. Sebagai contoh, awali perintah 
+dengan `sudo`, atau masuk sebagai `root` dan kemudian baru menjalankan perintah sebagai pengguna `root`.
+
+### _Driver_ cgroup
+
+Ketika systemd dipilih sebagai sistem init untuk sebuah distribusi Linux, proses init menghasilkan
+dan menggunakan grup kontrol root (`cgroup`) dan proses ini akan bertindak sebagai manajer cgroup. Systemd memiliki integrasi yang ketat
+dengan cgroup dan akan mengalokasikan cgroups untuk setiap proses. Anda dapat mengonfigurasi
+_runtime_ Container dan kubelet untuk menggunakan `cgroupfs`. Menggunakan `cgroupfs` bersama dengan systemd berarti
+akan ada dua manajer cgroup yang berbeda.
+
+_Control group_ digunakan untuk membatasi sumber daya yang dialokasikan untuk proses.
+Sebuah manajer cgroup tunggal akan menyederhanakan pandangan tentang sumber daya apa yang sedang dialokasikan
+dan secara bawaan (_default_) akan memiliki pandangan yang lebih konsisten tentang sumber daya yang tersedia dan yang sedang digunakan. Ketika kita punya memiliki 
+dua manajer kita akan memiliki dua pandangan tentang sumber daya tersebut. Kita telah melihat kasus di lapangan
+di mana Node yang dikonfigurasi menggunakan `cgroupfs` untuk kubelet dan Docker, dan `systemd`
+untuk semua sisa proses yang berjalan pada Node maka Node tersebut akan menjadi tidak stabil dibawah tekanan sumber daya.
```

Comment:
nit: add comma
```suggestion
untuk semua sisa proses yang berjalan pada Node, maka Node tersebut akan menjadi tidak stabil di bawah tekanan sumber daya.
```

Reply:
fixed in https://github.com/kubernetes/website/pull/22320/commits/3889465ad6b04da27431f82786a2a64ffe5ef7f1